docs/ckeditor.md

```diff
@@ -0,0 +1,24 @@
+# CKEditor
+
+We use the ckeditor in the dashboard in order for users to add formatted text, editable images, accordions and videos. To achieve this we have 4 configured editors for different use cases, default, image-editor, collapsible-image-editor, video-editor.
```

Comment:
Do we currently also use the default one without images for ideas?

Reply:
yep, we use image upload widget for images in ideas